docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -61,6 +61,15 @@ if set too high.
 
     Only applies for ``uniform`` :ref:`scheduler policy <node-scheduler-policy>`.
 
+``node-scheduler.max-unacknowledged-splits-per-task``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``2000``
+
+Maximum number of leaf splits not yet delivered to a given task.
```

Comment:
Could you improve the description here? Look at commit message of https://github.com/trinodb/trino/commit/3082ef8475f67132f0be0f82809d86460fb975ee